src/main/java/athena/TimeAllocator.java

```diff
@@ -28,6 +26,18 @@ public TimeAllocator(TaskList tasks) {
 
     }
 
+    public TimeAllocator() {
+
+    }
+
+    public void runAllocator() {
+        fixedTask.sort(new SortByStartTime());
```

Comment:
Missed this out last time. Since `fixedTask` and `flexibleTask` are lists, their name should be in plural form as per the coding standards (`fixedTasks` and `flexibleTasks`).